dependency-check-suppressions.xml

```diff
@@ -19,6 +19,8 @@
     <packageUrl regex="true">^pkg:javascript/handlebars\.js@.*$</packageUrl>
     <vulnerabilityName>Quoteless attributes in templates can lead to XSS</vulnerabilityName>
     <vulnerabilityName>poorly sanitized input passed to eval()</vulnerabilityName>
+    <vulnerabilityName>Disallow calling helperMissing and blockHelperMissing directly</vulnerabilityName>
```

Comment:
With both the stagemonitor removal and this swagger1 removal I think the handlebars and jquery suppressions can be completely removed. Once the stagemonitor PR is merged into a 4.x branch, you should be able to rebase this branch and confirm.

Reply:
bd24da2e